04_setup_ironic.sh

```diff
@@ -66,10 +66,19 @@ if [ ! -e "$IRONIC_DATA_DIR/html/images/$RHCOS_IMAGE_FILENAME_DUALDHCP" ] ; then
     sudo kpartx -d /dev/${LOOPBACK}
     sudo losetup -d /dev/${LOOPBACK}
     qemu-img convert -O qcow2 -c "$RHCOS_IMAGE_FILENAME_RAW" "$RHCOS_IMAGE_FILENAME_DUALDHCP"
+    ln -sf "$RHCOS_IMAGE_FILENAME_DUALDHCP" "$RHCOS_IMAGE_FILENAME_LATEST"
```

Comment:
We should probably move this outside the if statement, so it will work on existing environments where "$IRONIC_DATA_DIR/html/images/$RHCOS_IMAGE_FILENAME_DUALDHCP" exists?

Otherwise we get a 404 on deploy which I just hit.. :)

Reply:
> Otherwise we get a 404 on deploy which I just hit.. :)
doh, I was focusing on making sure the md5sum generation was going to work with existing deployments and forgot about this part...

Reply:
Ok I did this fixup and now works fine:

```
commit 67280ca7266c961450b6c1a89188265eb160db45
Author: Steven Hardy <shardy@redhat.com>
Date:   Tue Feb 26 10:03:50 2019 +0000

    fixup

diff --git a/04_setup_ironic.sh b/04_setup_ironic.sh
index 6f56ac7..17f87b2 100755
--- a/04_setup_ironic.sh
+++ b/04_setup_ironic.sh
@@ -66,10 +66,12 @@ if [ ! -e "$IRONIC_DATA_DIR/html/images/$RHCOS_IMAGE_FILENAME_DUALDHCP" ] ; then
     sudo kpartx -d /dev/${LOOPBACK}
     sudo losetup -d /dev/${LOOPBACK}
     qemu-img convert -O qcow2 -c "$RHCOS_IMAGE_FILENAME_RAW" "$RHCOS_IMAGE_FILENAME_DUALDHCP"
-    ln -sf "$RHCOS_IMAGE_FILENAME_DUALDHCP" "$RHCOS_IMAGE_FILENAME_LATEST"
     rm "$RHCOS_IMAGE_FILENAME_RAW"
     popd
 fi
+pushd $IRONIC_DATA_DIR/html/images
+ln -sf "$RHCOS_IMAGE_FILENAME_DUALDHCP" "$RHCOS_IMAGE_FILENAME_LATEST"
+popd

```

Reply:
thanks, I'm gonna move the other link creation aswell to keep them together.